_data/tools.yml

```diff
@@ -458,6 +458,23 @@
       url: https://www.elastic.co/elasticsearch/
     - urltext: Elasticsearch connector documentation
       url: https://trino.io/docs/current/connector/elasticsearch.html
+- name: FugueSQL
+  anchor: fugue-sql
+  category: client
+  logo: /assets/images/logos/fuguesql.png
+  logosmall: /assets/images/logos/fuguesql-small.png
+  description: |
+    Fugue provides an easier interface to using distributed compute effectively
+    and accelerates big data projects. It does this by minimizing the amount of
+    code you need to write, in addition to taking care of tricks and
+    optimizations that lead to more efficient execution on distrubted compute.
+    Fugue ports Python, Pandas, and SQL code to Spark, Dask, and Ray.
+  links:
+    - urltext: FugueSQL
+      url: https://fugue-tutorials.readthedocs.io/index.html
```

Comment:
Is there a specific page in their docs about Trino.. if yes .. add a link to that